File: 006.js
var _ = require('underscore');

function sumOfSquares(x) {
	var sum = 0;
	for(var i = 1; i <= x; i++) {
		sum += Math.pow(i, 2);
	}
	return sum;
}

function squareOfSum(x) {
	var sum = 0;
	for(var i = 1; i <= x; i++) {
		sum += i;
	}
	return Math.pow(sum, 2);
}

console.log(squareOfSum(100) - sumOfSquares(100));

File: 006.py
def sumOfSquares(x):
	sum = 0
	for i in range(1, x + 1):
		sum = sum + i**2
	return sum

def squareOfSum(x):
	sum = 0
	for i in range(1, x + 1):
		sum = sum + i
	return sum**2

print squareOfSum(100) - sumOfSquares(100)